gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -1368,7 +1368,9 @@ function plymeta:SafePickupWeapon(wep, ammoOnly, forcePickup, dropBlockingWeapon
 		-- Very very rarely happens but definitely breaks the weapon and should be avoided at all costs
 		if dropWeapon == wep then return end
 
-		self:SafeDropWeapon(dropWeapon, true)
+		timer.Simple(0, function()
```

Comment:
please add a `if not IsValid(self) or not IsValid(dropWeapon) then return end` check here to prevent further errors. Otherwise it looks good to me

Reply:
Added